apollo-runtime/src/main/java/com/apollographql/apollo/cache/normalized/sql/SqlStore.java

```diff
@@ -86,6 +89,10 @@ private SqlStore(ApolloSqlHelper dbHelper, FieldsAdapter parser) {
     return changedKeys;
   }
 
+  @Override public void clearAll() {
```

Comment:
We should not forget to add javadoc and mention that this function shouldn't be called directly from client only via Cache